commitlint.config.ts

```diff
@@ -28,17 +28,17 @@ module.exports = {
         'footer-leading-blank': [RuleStatus.Warning, 'always'],
         'footer-max-line-length': [RuleStatus.Error, 'always', 150],
         'header-max-length': [RuleStatus.Error, 'always', 50],
+        'prefer-slash-over-backslash': [RuleStatus.Error, 'always'],
         'subject-full-stop': [RuleStatus.Error, 'never', '.'],
         'type-empty': [RuleStatus.Warning, 'never'],
         'type-space-after-colon': [RuleStatus.Error, 'always'],
         'subject-lowercase': [RuleStatus.Error, 'always'],
         'type-space-after-comma': [RuleStatus.Error, 'always'],
         'trailing-whitespace': [RuleStatus.Error, 'always'],
-        'prefer-slash-over-backslash': [RuleStatus.Error, 'always'],
```

Comment:
@realmarv when rebasing, you deleted this by mistake; this line needs to be reinstated before I can merge this PR

Reply:
It's in line 31. I thought rules should be ordered alphabetically, but now that I know it's based on their priority, should I put it at the end of the list?

Reply:
we were writing tests in alphabetical level; then in another PR I pointed out that rules could be in alphabetical level too; but we never talked about configuration of rules being in alphabetical order

Reply:
typo: level -> order

Reply:
Done!